langgraph/store/redis/__init__.py

```diff
@@ -335,16 +389,21 @@ def _batch_put_ops(
                         "updated_at": datetime.now(timezone.utc).timestamp(),
                     }
                 )
-                vector_key = f"{STORE_VECTOR_PREFIX}{REDIS_KEY_SEPARATOR}{doc_id}"
-                vector_keys.append(vector_key)
+                redis_vector_key = f"{STORE_VECTOR_PREFIX}{REDIS_KEY_SEPARATOR}{doc_id}"
```

Comment:
`vector_key` was defined just above here as `vector_key: tuple[str, str] = (ns, key)` and used for something else.